XenModel/XenAPI-Extensions/VM.cs

```diff
@@ -683,7 +683,14 @@ void VirtualizationTimer_Elapsed(object sender, ElapsedEventArgs e)
         private Timer VirtualizationTimer = null;
 
         [Flags]
-        public enum VirtualisationStatus { UNKNOWN = 0, OPTIMIZED = 1, PV_DRIVERS_NOT_INSTALLED = 2, PV_DRIVERS_OUT_OF_DATE = 4 };
+        public enum VirtualisationStatus
+        {
+            PV_DRIVERS_NOT_INSTALLED    = 0,

```

Comment:
Because PV_DRIVERS_NOT_INSTALLED has the value zero, it isn't really a flag like the others, and it might be better just to delete it. I can see that it's sometimes nice to use a symbolic name, but I think it is likely lead to bugs like the one above, and it would be better for people to test for the absence of IO_DRIVERS_INSTALLED and/or MANAGEMENT_INSTALLED.